protocol/src/main/java/org/apache/drill/exec/proto/beans/CoreOperatorType.java

```diff
@@ -131,6 +139,14 @@ public static CoreOperatorType valueOf(int number)
             case 43: return HIVE_DRILL_NATIVE_PARQUET_ROW_GROUP_SCAN;
             case 44: return JDBC_SCAN;
             case 45: return REGEX_SUB_SCAN;
+            case 46: return MAPRDB_SUB_SCAN;
```

Comment:
I think for future we should come up with a strategy to avoid missing OperatorType from protobuf and also probably separate the contrib related format plugins or operator to have a different range of values. This will help different distro's to also define their own operator in that specific range rather than having conflict with Drill's internal core operators. Having different range is still a short term solution.

For longer term we can discuss other solutions like some form of registration techniques for all format plugin and operator which will advertise their own unique id (thanks to @parthchandra) for suggesting it. But we can discuss more for future improvements.

Reply:
Good point, also once I have created Jira to allow operators names to be in String rather with dependency on protobuf but this would be a breaking change (https://issues.apache.org/jira/browse/DRILL-5405).